qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -59,11 +63,14 @@ def __init__(
                 Only enforced when not using the state vector simulator. Default True.
             batch_size: Number of circuits to batch together for computation. Default 1000.
             quantum_instance: Quantum Instance or Backend
+            free_parameters: Iterable containing Parameter objects which correspond to
+                free parameters (non-data-bound parameters) in the feature map circuit
```

Comment:
Can we say something more here in this docstring to better explain them. I imagine whatever these free parameters are that the user of this class needs to set their values ahead of the kernel being evaluated so in this case the user of any quantum kernel instance ends up being responsible for them - to that end they are more like 'user parameters' in that regard.